espn_api/requests/espn_requests.py

```diff
@@ -73,6 +75,10 @@ def get_league(self):
             'view': ['mTeam', 'mRoster', 'mMatchup', 'mSettings', 'mStandings']
         }
         data = self.league_get(params=params)
+        # Check if the data is a list (which happens when year is 2018 or earlier)
+        if isinstance(data, list):

```

Comment:
this check can be moved to league_get call so that it only has to be done once. league_get is already doing this for any year before 2018. That logic can be updated with this check instead

Reply:
This makes sense, I missed that the first go around. Updated the logic there & removed the check for league year.